tests/acceptance/acceptance_test.go

```diff
@@ -80,6 +80,16 @@ var _ = Describe("Acceptance", func() {
 			// not register as running, even though all child resources are fine
 			// :shrug_emoji:
```

Comment:
think this comment can go since now helm release is fine

Reply:
:+1: I'll update the test to check the helm-release status too